### PR TITLE
docs(react-tinacms-date): Typo fix: s/Craeted/Created/

### PR DESCRIPTION
**Pull request by notjosh**
✍️✅

### CODE REVIEW

**Review by DirtyF — APPROVED** [review]

**Comment by DirtyF** [comment]
@notjosh Please sign the CLA 🙏

**Comment by notjosh** [comment]
Ah, that's a lot of legalese for such a trivial change :[

I'll leave it to an existing maintainer to fix. Thanks anyway!

**Pull request closed without merging**
